data/hai/unfettered jobs.txt

```diff
@@ -322,3 +401,408 @@ mission "Unfettered Big Medicals Rush"
+	on complete
+		outfit "Ion Cannon" -2
+		payment 600000
+		dialog `You go bring the Ion Cannons personally to the captain, "Thanks, I will bring those to my warlord as soon as possible. Here is an extra <payment> for the journey. I wish you safe travels!" She seems in a bit of a rush, and leaves straight away.`
+
+
+
+phrase "jump drive or stopover"
+	word
+		`You have a mission to bring jump drives to <planet>, from <stopovers>. You may have left your escort allocated to carrying the devices behind, or you have not retrieved the precious cargo yet.`
+
+phrase "unfettered jump drive pickup dialog"
+	word
+		"You manage to land and retrieve the precious cargo without too much trouble. Do not rely on luck too long though - you should depart quickly!"
+		"You find a few pirates in the vault, but they are quickly disposed of. You proceed to pick up the cargo; do not wait too long, or reinforcements could arrive."
+		"It was easy - almost too easy - to pick up the jump drives. Better depart before your luck runs out."
+		"You receive a warning from the vault about an intrusion, but cannot find anyone. The cargo is swiftly transferred into your ship, and you should probably get on your way now."
+		"The vault is secured, and you can easily and safely pick up the jump drives."
+		"Given how hard it can be to collect these jump drives, you sometimes wonder if they actually belong to the Unfettered."
+
+phrase "unfettered jump drive payment dialog"
+	word
+		"The Unfettered are more than content with paying <payment> in exchange for your precious cargo."
+		"The Unfettered are in a rush to get the devices, hoping to install them on their ships. They give you <payment>."
+		"Every time you bring jump drives back, it seems to be a cause for celebration for the Unfettered. This time is no different, and they almost forget to give you the <payment>."
+		"Your cargo is unloaded, and the Unfettered give you <payment> for it. Curious about the source, you pay attention to a conversation between two members of the same crew and hear something about unfettered humans."
+		"After unloading the cargo, you try your luck asking about who places these jump drives you're collecting, but, as usual, the Unfettered workers give you no information about it. Fortunately, they still give you <payment>."
+
+mission "Unfettered Get Jump Drives(1)"
+	job
+	repeat
+	name "Collect a jump drive"
+	description "Go to the pirate world of <stopovers> to recover a jump drive from a vault, then bring it to <destination> for <payment>."
+	source
+		attributes "unfettered"
+	stopover
+		distance 12 18
+		government "Pirate"
+	clearance "When you tell the pirates you are here on behalf of the Unfettered, they immediately allow you to land without asking any questions."
+	destination "Darkcloak"
+	to offer
+		random < 30
+		has "Unfettered: Prove your strength 2: done"
+	on stopover
+		dialog phrase "unfettered jump drive pickup dialog"
+		outfit "Jump Drive"
+		"spawn jd fleet 1"++
```

Comment:
```suggestion
		"spawn jd fleet 1" ++
```
I think this one works? Idk, I see your version used in multiple missions here.

Reply:
++ or space then ++ is the same yeah